homework_8/geometricFigure/GeometricFigure.java

```diff
@@ -0,0 +1,13 @@
+package geometricFigure;
+
+public interface GeometricFigure {
+    double getArea();
+
+    static void summAreas(GeometricFigure[] geometricFigures) {
```

Comment:
this method should return a value, precisely, a total area of all figures, so instead of `void` return `double`